src/migrations/0010_amusing_kingpin.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `agents` ADD `tools` text DEFAULT '["computer", "goal_solution", "publications", "system_prompt_self_edit"]' NOT NULL;
```

Comment:
This will break existing experiment (which is fine) but code wise the default should be aligned withg DEFAULT_TOOLS so no computer

Reply:
But that migration is still off ?